companies/fkb.json

```diff
@@ -0,0 +1,20 @@
+{
+    "slug": "fkb",
+    "relevant-countries": [
+        "li"
+    ],
+    "categories": [
+        "health"
+    ],
+    "name": "FKB – Die liechtensteinische Gesundheitskasse",
+    "address": "Gagoz 75\nPostfach 363\n9496 Balzers\nLiechtenstein",
+    "phone": "+423 388 19 90",
+    "fax": "+423 388 19 91",
+    "email": "info@fkb.li",
```

Comment:
> oder unter info@fkb.li - Betreff beginnend mit „[Datenschutzbeauftragte]:“ - erreichbar. 

You could add this as a comment. @baltpeter Do we want to support something like this in the long run? :D

Reply:
No. And I don't think it's necessary to add that as a comment either. Our subjects should already be more than clear enough. And they aren't legally allowed to force you to add that to the subject.

Reply:
Ok!